src/components/volunteer/VolunteerSteps2.jsx

```diff
@@ -0,0 +1,301 @@
+            target="_blank"
+            rel="noreferrer"
+            style={{
+              color: '#56B6A1',
+              textDecoration: 'none',
+              fontWeight: 600
+            }}
+          >
+            HERE
+          </a>
+        </Typography>
+      </>
+    )
+  },
+  {
+    label: 'Choose a Role & Adopt Our Standards',
+    description: (
+      <>
+        <ul>
+          <li>
+            After seeing what we are up to, reach out to team members of a project to connect &amp;
+            let them know how you would like to contribute. You can find active members by seeing
+            who recently contributed to a repository or the recent discord chat history of a
+            project.
+          </li>
+          <li>
+            The GitHub repository is a good place to see what&apos;s active within CODE PDX and
+            contains a writeup of our contribution guidelines that we follow to standardize our
+            workflow.
+          </li>
+        </ul>
+      </>
+    )
+  },
+  {
+    label: 'Contribute',
+    description: (
+      <>
+        <ul>
+          <li>
+            Find your first issue to work on from the GitHub repo of the project of your choice.
+            Check out our{' '}
+            <a
+              style={{
+                color: '#56B6A1',
+                textDecoration: 'none',
+                fontWeight: 600
+              }}
+            >
+              Wiki
+            </a>{' '}
+            for guidance on picking your first issue.
+          </li>
+          <li>Create a branch for your issue or feature and collaborate with other members.</li>
+          <li>
+            Create a pull request off of our GitHub template, and submit for review. Pull requests
+            require at least one other person to review & approve before merging.
+          </li>
+        </ul>
+      </>
+    )
+  },
+  {
+    label: 'Additional Onboarding Info',
+    description: (
+      <>
+        <ul>
+          <li>All of these steps can be completed remotely or at an in-person event.</li>
+          <li>
+            We invite people to contribute to CODE PDX regardless of skill set or skill level.
+          </li>
+          <li>
+            When in doubt, proactively reach out in the Discord text channels. They are frequently
+            checked by members and will likely yield a swift response.
+          </li>
+        </ul>
+      </>
+    )
+  }
+];
+
+const VolunteerSteps2 = () => {
+  const [activeStep, setActiveStep] = useState(0);
+
+  const handleNext = () => {
+    setActiveStep((prevActiveStep) => prevActiveStep + 1);
+  };
+
+  const handleBack = () => {
+    setActiveStep((prevActiveStep) => prevActiveStep - 1);
+  };
+
+  const handleReset = () => {
+    setActiveStep(0);
+  };
+
+  return (
+    // <Container maxWidth="md">
+    <Box sx={{ width: '100%' }}>
+      <Stepper activeStep={activeStep} orientation="vertical">
```

Comment:
Frankly I'm not a huge fan of the `Stepper` version (shown in Desktop 2), especially for large viewports where there's plenty of space. The Desktop 1 version looks cleaner and more intuitive in my opinion. 

To expand on this, I think the less input we require from the user the better. 

Personally, I would be slightly frustrated if I had to click through multiple times to access information that could have been presented all at once. Especially because there isn't any other unique content on this page, so it's not competing for real estate or anything. I'm thinking the same for the mobile version. No real need to consolidate there.

For better or for worse, I think we have to take into account some users being impatient.